packages/project-config/build.js

```diff
@@ -0,0 +1,26 @@
+import * as esbuild from 'esbuild'
+
+const optiions = {
```

Comment:
```suggestion
const options = {
```

Reply:
Fixed in https://github.com/redwoodjs/redwood/pull/9870/commits/07667b20f4791cdae3c41d6c46c150397e6cc21d